### PR TITLE
chore(web): onboarding animations update

### PR DESCRIPTION
**Pull request by LetItRock**
### What change does this PR introduce?

Onboarding Improvements:
- updated the use-case animations
- added the static image for the Translation use-case

### Other information (Screenshots)

https://github.com/novuhq/novu/assets/2607232/458993e4-6b3e-42df-9194-01193d662758

### CODE REVIEW

File: apps/web/src/pages/get-started/components/GetStartedAnimation.tsx [hunk 1]
```diff
@@ -11,15 +13,29 @@ interface IGetStartedAnimationProps {
 // uses `public` as the default base directory
 const ROOT_ANIMATION_PATH = `animations/get-started`;
 const STATE_MACHINE_NAME = 'SM';
-const STATE_MACHINE_INPUT_NAME = 'white theme';
+const STATE_MACHINE_INPUT_NAME = 'theme';
+const LIGHT_THEME_INPUT = 2;
+const DARK_THEME_INPUT = 0;
```
Comment (BiswaViraj) [inline apps/web/src/pages/get-started/components/GetStartedAnimation.tsx:18]: Can we add a comment about these values. why it needs to be 0 and 2 and not 0, 1?

**Review by BiswaViraj — CHANGES_REQUESTED** [review]
Looks good 🚀 
Have added a small comment

File: apps/web/src/pages/get-started/components/GetStartedAnimation.tsx (hunk 1 above)
Comment (LetItRock) [reply apps/web/src/pages/get-started/components/GetStartedAnimation.tsx]: left a comment

**Review by BiswaViraj — APPROVED** [review]
🚀

**Pull request merged**